src/admin-views/editor/panel/fields/dates.php

```diff
@@ -65,7 +65,6 @@ class="tribe-timepicker tribe-field-start_time ticket_field"
 			aria-label="<?php echo esc_attr( $ticket_start_date_aria_label ); ?>"
 		/>
 		<span class="helper-text hide-if-js"><?php esc_html_e( 'HH:MM', 'event-tickets' ); ?></span>
-		<span class="dashicons dashicons-editor-help" title="<?php echo esc_attr( $ticket_start_date_help_text ); ?>"></span>
```

Comment:
I don't see any other usages of this `$ticket_start_date_help_text` in this template. So, we should remove it completely from the template including the places where it is [defined](https://github.com/the-events-calendar/event-tickets/blob/3b54820652d7a7c9a4e0116da108eb476c0ff933/src/Tickets/Admin/Panels_Data/Ticket_Panel_Data.php#L120)

Reply:
The same for the end date helper text.

Also, we should update the template version and since version for the changes made.

Reply:
Worth doing a pass through all of them and 
1. removing them from the file docblock
2. adding them to an `@since` as removed